### DISCUSSION TITLE
Added: LDM Notes for April 12th, 2021

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#4646] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-04-12.md

## Agenda

1. List patterns
2. Lambda improvements

**Comment by cheong00** [comment]
For some reason I'm more interested in what is the [redacted] part in "Quote(s) of the day" than the actual discussions. :stuck_out_tongue_winking_eye:

**Reply by alrz** [reply]
"Quote of the day" has become more like "LDM out of context"

**Reply by 333fred** [reply]
Redacted was a name.

**Comment by Joe4evr** [comment]
> `async async (async async) => async`

LDM channeling their inner C++, now. 🙃

**Reply by bernd5** [reply]
I love contextual keywords 😅

**Comment by yaakov-h** [comment]
Just for clarity on the async async scenario, does the escaping discussion mean that the code would have to become this?:

```csharp
// async return-type (param-type param-name) => expression
   async @async      (async      async     ) => async
```

**Reply by 333fred** [reply]
Yes.

**Reply by MgSam** [reply]
I, for one, vote for the compiler to reformat your hard disk if you try to write code like this.
